extensions/amp-story/1.0/amp-story.js

```diff
@@ -2003,7 +1995,7 @@ export class AmpStory extends AMP.BaseElement {
   }
 
   /**
-   * @return {number}
+   * @return {number} Number of story pages excluding ad pages.
```

Comment:
This method looks unused, you can delete it and its test.

Reply:
Done